### ISSUE TITLE
Installation fails due to invalid CRIO registries

### ISSUE THREAD

**Issue by nadenf** [Kubeinit/kubeinit#471]
**Describe the bug**
Installation fails due to invalid CRIO registries

**To Reproduce**
1. Clone git@github.com:Kubeinit/kubeinit.git
2. Run instructions on https://github.com/Kubeinit/kubeinit

**Infrastructure**
 - Hypervisors OS: Ubuntu 21.04

**Deployment command**
```
ansible-playbook \
    --user root \
    -v -i ./hosts/eks/inventory \
    --become \
    --become-user root \
    ./playbooks/eks.yaml
```

**Inventory file diff**

```
< hypervisor-01 ansible_host=harana
---
> hypervisor-01 ansible_host=nyctea
```

**Comment by gmarcy** [comment]
@nadenf can you check if this is still an issue?

**Comment by ccamacho** [comment]
I reproduced this, this should be closed by #483
